legal-api/src/legal_api/models/corp_type.py

```diff
@@ -0,0 +1,57 @@
+# Copyright © 2021 Province of British Columbia
```

Comment:
this is a very slow moving set of data.
Any changes to it probably require the test suite to run.
and sine there's no tracking to say if Lear manages this entity type yet, could it just be a set of Enum & Dicts?

Reply:
It could be but the requirement says we need a table with all entity type listed in the attached doc. Maybe we can include a new column to track `lear_ind`. But do we validate this now?